s5_subwords/run.sh

```diff
@@ -160,17 +172,35 @@ echo ===========================================================================
 echo "                		Prepare LM with subword text files with $method          "
 echo ============================================================================
 
-  local/lm/prepare_lm_subword.sh $subword_dir/text_corpus \
-                              data/$lang/test/text \
-                              data/$lang/local/dict/lexicon.txt \
-                              data/$lang/local/lm \
-                              6               
-
-  utils/format_lm.sh  data/$lang/lang \
-                      data/$lang/local/lm/lm.gz \
-                      data/$lang/local/dict/lexicon.txt \
-                      data/$lang/lang_test
-fi
+  if [ $lm_tool == 'srilm' ]; then
+    local/lm/prepare_lm_subword.sh $subword_dir/text_corpus \
+                                data/$lang/test/text \
+                                data/$lang/local/dict/lexicon.txt \
+                                data/$lang/local/lm \
+                                6               
```

Comment:
Why do you use lang/local for the lms? Is it not best, further down the line, to know that the lm will be in $lang independent of which tool was used to create it?